controller/OrderController.go

```diff
@@ -0,0 +1,107 @@
+package controller
+
+import (
+	"fmt"
+	"github.com/Baraulia/COURIER_SERVICE/dao"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+var Orders []dao.Order
+
+func (h *Handler) GetCourierCompletedOrders(c *gin.Context) {
+	page, er := strconv.Atoi(c.Query("page"))
+	if er != nil || page == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"message": " expect an integer"})
+		return
+	}
+	limit, er1 := strconv.Atoi(c.Query("limit"))
+	if er1 != nil || limit == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"message": " expect an integer"})
+		return
+	}
+	idCourier, er := strconv.Atoi(c.Query("idcourier"))
+	if er != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"message": " expect an integer"})
+		return
+	}
+
+	Orders, err := h.services.OrderApp.GetCourierCompletedOrders(limit, page, idCourier)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"message": fmt.Sprintf("Error: %s", err)})
+		return
+	}
+	c.JSON(http.StatusOK, Orders)
+
+}
+
+func (h *Handler) GetAllOrdersOfCourierService(c *gin.Context) {
+	page, er := strconv.Atoi(c.Query("page"))
+	if er != nil || page == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"message": " expect an integer"})
+		return
+	}
+	limit, er1 := strconv.Atoi(c.Query("limit"))
+	if er1 != nil || limit == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"message": " expect an integer"})
+		return
+	}
+	idService, er := strconv.Atoi(c.Query("iddeliveryservice"))
+	if er != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"message": " expect an integer"})
+		return
+	}
```

Comment:
🔴 All query parameters are required. But I don't see any docs describing it.

You should provide users a possibility to make simple API calls like `GET /orders/completed` without query params.

Reply:
Will it be enough to add a swagger? Or is it necessary to provide simple implementation?

Reply:
Swagger would be enough, but it better is to provide pure implementation.